packages/blade/.storybook/react/manager.js

```diff
@@ -41,6 +42,47 @@ export const theme = {
   // brandImage: 'https://place-hold.it/350x150',
 };
 
+const ADDON_ID = 'internal-components-addon';
+const TOOL_ID = 'internal-components-tool';
+const hiddenStoryStyle = document.createElement('style');
+hiddenStoryStyle.textContent = `
+  [id*='internal'] {
+    display: none !important;
+  }
+`;
+document.head.append(hiddenStoryStyle);
+
+const InternalStoryAddon = () => {
+  const [{ showInternalComponents }, updateGlobals] = useGlobals();
+
+  const toggleVisibility = React.useCallback(() => {
+    updateGlobals({
+      showInternalComponents: !showInternalComponents,
+    });
+    hiddenStoryStyle.disabled = showInternalComponents ? undefined : 'disabled';
```

Comment:
what does `disabled` do here?

Reply:
It disables the stylesheet 
https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet/disabled